src/workerd/io/BUILD.bazel

```diff
@@ -351,13 +352,17 @@ kj_test(
 
 kj_test(
     src = "promise-wrapper-test.c++",
-    deps = [":io"],
+    deps = [
+        ":io",
+        "//src/workerd/api:urlpattern",
+    ],
 )
 
 kj_test(
     src = "compatibility-date-test.c++",
     deps = [
         ":io",
+        "//src/workerd/api:urlpattern",
```

Comment:
Why does `compatibility-date-test` depend on urlpattern? 🤔

Reply:
I have no idea but I'm investigating the weird linker errors right now